plugins/w3c_logs.yaml

```diff
@@ -104,10 +146,21 @@ template: |
       operators:
         - type: filter
           expr: 'body matches "^#"'
-
         - type: csv_parser
           ignore_quotes: true
+          parse_from: '{{ .parse_from }}'
+          parse_to: {{ .parse_to }}
+          lazy_quotes: {{ .lazy_quotes }}
+          ignore_quotes: {{ .ignore_quotes }}
```

Comment:
looks like `ignore_quotes` is defined twice here.

Reply:
Per your reference to the spec, I've removed the quotes options.